console/frontend/src/main/frontend/src/app/app-routing.module.ts

```diff
@@ -141,6 +141,22 @@ const routes: Routes = [
       breadcrumbs: 'Logging > Log Settings',
     },
   },
+  {
+    path: 'logging/:directory',
+    component: LoggingComponent,
+    title: 'Logging',
+    data: {
+      breadcrumbs: 'Logging > Log Files',
+    },
+  },
+  {
+    path: 'logging/:directory/:file',
+    component: LoggingComponent,
+    title: 'Logging',
+    data: {
+      breadcrumbs: 'Logging > Log Files',
```

Comment:
```suggestion
      breadcrumbs: 'Logging > Show Directory (/name) > Show File (name.png)',
```

Reply:
This should be possible by using `AppService.customBreadcrumbs(breadcrumbs: string): void` in the initialisation of the component, this will overwrite the current breadcrumbs until the user switches to a different page or the function is called again.

Reply:
Thank you for this recommendation. I've implemented it.